app/facades/quotable_facade.rb

```diff
@@ -0,0 +1,11 @@
+class QuotableFacade < QuotableService
+  def self.quote_search(keyword)
```

Comment:
Good thinking here to get both the quote info that you want as well as the `totalCount`. You can just build out a single method and pass it those two things. An example:
```
def self.quotes(search_phrase)
    quote_data = QuotesService.get_quotes(search_phrase)
    quotes = quote_data[:results].first(10).map do |quote|
      QuoteSearchResult.new(quote)
    end
    [quote_data[:totalCount], quotes]
  end
```
You would only need to do the`.first(10)` if you didn't set a param to limit in your service.